sympy/core/mul.py

```diff
@@ -1328,6 +1336,17 @@ def _eval_is_algebraic(self):
             return self.is_zero
 
     def _eval_is_zero(self):
+        n, d = fraction(self, exact=True)
+        if not d.is_Integer:
+            if n.is_zero:
+                if d.is_zero is False:
+                    return True
+            if d.is_finite is None:
+                return
+            if n.is_zero is False:
+                return False
```

Comment:
This gives:
```python
In [5]: n = Symbol('n', extended_nonzero=True)

In [6]: d = Symbol('d', finite=False)

In [7]: (n/d).is_zero
Out[7]: False
```
We don't know though if this is `oo/oo` so it should give `None`.